es6-async-await/await.js

```diff
@@ -0,0 +1,68 @@
+import fetch from './fetch.js';
+
+// Keep track of app load time so each log message can be timed.
+// Log messages should all be approximately 1 second apart.
+const startTime = Date.now();
+const elapsed = () => `${Math.round((Date.now() - startTime) / 1000)}s -`;
+
+async function fetchOnce() {
+  try {
+    const response = await fetch('foo/bar.html');
+    const msg = await response;
```

Comment:
You already `await`ed in line 10, so there's no reason to `await` again here.
You could just say `const msg = await fetch('foo/bar.html');`